docs/_articles/en/lwc/linting.md

```diff
@@ -11,37 +11,47 @@ Linting provides errors about malformed code while you edit. VS Code enforces Sa
 
 ## Installation
 
-1. If you created an SFDX project using the `force:project:create` command, the project may contain a `package.json` file with the ESLint plugin already included. Open `package.json` and verify that it contains the following:
+### For a New Project
 
-   ```json
-   "scripts": {
-       "lint": "npm run lint:lwc",
-       "lint:lwc": "eslint force-app/main/default/lwc"
-   }
-   ```
+If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included.
 
-2. If your `package.json` does not contain the above lint commands, run `npm install @salesforce/eslint-config-lwc eslint@5 -D`.
+1. To install the ESLint plugin and other dependencies, run `npm install` in your project directory.
 
-3. To start linting, run `npm run lint:lwc`. You must have components in your project in order to run this command.
+2. Copy the [.eslintrc.json](https://github.com/forcedotcom/salesforcedx-templates/blob/master/src/templates/project/.eslintrc.json) file and save it to your project directory. Configure the ESLint plugin rules as needed.
 
-4. If you don't have a `package.json` file, copy [this file](https://github.com/forcedotcom/salesforcedx-templates/blob/master/src/templates/project/package.json) and save it in your project. Then, run `npm install`.
+3. To run linting, you must have components in your project. To start linting, run `npm run lint:lwc`.
 
-5. Check whether you have `.eslintignore` and `.eslintrc.json` files included in your project. 
-    - If you have `.eslintrc.json`, add the following line to the file.
+### For an Existing Project
 
-         ```json
-         {
-          "extends": ["@salesforce/eslint-config-lwc/recommended"]
-         }
-         ```
+1. Verify that your project has a `package.json` with these configurations. If it doesn't, If it doesn't, run `npm install @salesforce/eslint-config-lwc eslint@5 -D`.
 
-    - If your project doesn't have either file, copy the files to your project from the [project templates](https://github.com/forcedotcom/salesforcedx-templates/tree/master/src/templates/project) Github repository.
+    ```json
```

Comment:
Running the npm install won't add the scripts to their package.json.

Reply:
What does that command do? Nate wanted me to add it.

Reply:
npm install adds the dependencies. But you still have to manually copy the scripts over.